tests/sls/homu/valid_builders.py

```diff
@@ -0,0 +1,40 @@
+import json
+import subprocess
+import toml
+
+from tests.util import Failure, Success
+
+
+def is_builders_valid(buildbot, homu):
+    for builder in homu:
+        if builder not in buildbot:
+            return False
+    return True
+
+
+def run():
+    homu_cfg = toml.load('/home/servo/homu/cfg.toml')
+    homu_buildbot = homu_cfg['repo']['servo']['buildbot']
+
+    ret = subprocess.run(
+        ['python2', 'tests/sls/homu/get_buildbot_cfg.py'],
```

Comment:
You'll need to use `/usr/bin/python2`. This is because Travis's only has one "mega" Trusty image, on which it installs all its Pythons, Rubies, etc., and the `$PATH` always contains all of those Pythons before the main `/usr/bin`, etc. entries. Hence, just `python2` invokes a Python from `/opt`, not the main Python that we want.